package/docker/scripts/reports/sql/NumberOfPatients.sql

```diff
@@ -0,0 +1,15 @@
+SELECT count(*) as 'count' , 'old' as 'type'
+FROM `visit`
```

Comment:
any specific reason to put 'quotes' for table/column names?

Reply:
No specific reason. I'll remove them